packages/babel-helpers/src/helpers/usingCtx.ts

```diff
@@ -31,11 +31,23 @@ export default function _usingCtx() {
         var dispose =
           value[Symbol.asyncDispose || Symbol.for("Symbol.asyncDispose")];
       }
-      if (dispose == null) {
+      if (dispose === undefined) {
         dispose = value[Symbol.dispose || Symbol.for("Symbol.dispose")];
+        if (isAwait) {
+          var inner = dispose;
+        }
```

Comment:
`var inner = isAwait && dispose;`
Will this be smaller?

Reply:
```diff
- // size: 972, gzip size: 498
+ // size: 971, gzip size: 500
```
It will save one byte for plaintext but the compressed size is increased by 2.

Reply:
```diff
- // size: 972, gzip size: 498
+ // size: 971, gzip size: 500
```
It will save one byte for plaintext but the compressed size is increased by 2.